templates/swift/Sources/Client.swift.twig

```diff
@@ -228,7 +229,7 @@ open class Client {
         headers: [String: String] = [:],
         params: [String: Any?] = [:],
         sink: ((ByteBuffer) -> Void)? = nil,
-        convert: (([String: Any]) -> T)? = nil,
+        converter: (([String: Any]) -> T)? = nil,
```

Comment:
Why this change?

Reply:
Just a clearer name for the parameter, it is only used internally